Src/AutoFixture.xUnit2.UnitTest/MemberAutoDataAttributeTest.cs

```diff
@@ -275,6 +177,17 @@ public void PreDiscoveryShouldBeDisabled()
             Assert.Equal(expectedAssembly, actualAssembly);
         }
 
```

Comment:
Shouldn't there also be a test that verifies that `MemberAutoDataAttribute.GetData` returns the values from `FooBarTestCase` and in the correct order?

Reply:
Hi @ecampidolgio, I was under the impression the `Scenario.cs` was the place holder for all the tests around data. It has a few tests to verify the data produced by the `MemberAutoDataAttribute`, which I believe may cover what you suggested.

Please let me know if you agree or you still want me to add more tests.

Reply:
Hi @ecampidoglio. Would you be able to provide your blessing or reprimand on the above ? :)

Reply:
You're right that the tests in `Scenario` verify the behavior; however, those are _end-to-end tests_ and, as such, they may fail for a number of different reasons. Adding a focused unit test for just this part would help narrow down any potential problems.